h2o-genmodel/src/main/java/hex/genmodel/attributes/parameters/Pair.java

```diff
@@ -0,0 +1,6 @@
+package hex.genmodel.attributes.parameters;
+
+public interface Pair<K, V> {
```

Comment:
not needed, you can always generalize later - currently there is no need to do that

this interface is actually harmful - it is making you do autoboxing of Doubles to double in your API